### PR TITLE
added en.po file

### PR DESCRIPTION
**Pull request by patogit**
don't know why it wasn't there. Just copied the msgid to the msgstr.

### CODE REVIEW

**Comment by p4u** [comment]
I might not understand something but  why is a po English file needed if Enflish is the main/reference language which is written in the LUA code.

https://github.com/libremesh/lime-packages/blob/develop/packages/lime-webui/luasrc/model/essentials.lua

**Comment by patogit** [comment]
Without a .PO or .POT file in English, the translation interface won't work.  We need that file in order for the translation platform to show the strings that we will translate, i.e. the source file.

But maybe me adding it by hand isn't the proper way? Maybe it's supposed to be done with a script, in order to keep everything up to date?

**Comment by patogit** [comment]
See quoted request for this file at https://github.com/libremesh/lime-packages/issues/257#issuecomment-367206873

**Comment by patogit** [comment]
@p4u , any action on this? It's one of the last pieces we're waiting on in order to set up the space at TranslateWiki.

**Comment by Nikerabbit** [comment]
I am not sure whether this file will work. It seems to lack the normal gettext headers. Compare with [this](https://github.com/mantisbt-plugins/codev/blob/master/i18n/locale/codevtt.pot#L6-L18) for example (also msgstr is empty in pot files).

**Comment by patogit** [comment]
I just updated to repo. Deleted the po/en/lime-webui.po and added po/lime-webui.pot with headers as indicated by @Nikerabbit . (Sorry for the delay, my work sometimes takes me out of Internet connectivity.)

**Comment by patogit** [comment]
Now I'm reading the docs on how to generate the POT file autmatically... https://gluon.readthedocs.io/en/v2017.1.x/dev/web/i18n.html#adding-translation-templates-to-gluon-packages

**Comment by patogit** [comment]
updated strings using script, and then re-inserted the headers. I wonder how to make the script insert the headers.

File: packages/lime-webui/po/lime-webui.pot [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
-"Project-Id-Version: \n"
+"Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2018-03-19 21:44-0500\n"
+"POT-Creation-Date: 2018-03-18 13:49+0100\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: en_US\n"
+"Language: it_IT\n"
```
Comment (altergui) [inline packages/lime-webui/po/lime-webui.pot]: why it_IT?

**Comment by altergui** [comment]
err, i mean the three commits can be squashed together (the first two commits are pointless)

File: packages/lime-webui/po/lime-webui.pot (hunk 1 above)
Comment (patogit) [reply packages/lime-webui/po/lime-webui.pot]: It's just a template. That language code will get changed in each real translation file. it_IT is just an example.

**Comment by patogit** [comment]
I don't know how to squash those three commits, and the tutorials I'm trying aren't working.

File: packages/lime-webui/po/lime-webui.pot [hunk 2]
```diff
@@ -0,0 +1,248 @@
+msgid ""
```
Comment (Nikerabbit) [inline packages/lime-webui/po/lime-webui.pot]: Pot files should start with `#, fuzzy`

**Comment by Nikerabbit** [comment]
One can "Squash and merge" with GitHub directly.

Also, can you clarify whether this file is generated by script or manually? I think there must be a script that keeps this file up to date when strings are changed or you won't be getting translations for the new strings.

File: packages/lime-webui/po/lime-webui.pot (hunk 2 above)
Comment (patogit) [reply packages/lime-webui/po/lime-webui.pot]: Ok, I added that in a commit just now.

**Comment by patogit** [comment]
I don't see any "Squash & merge" button on GitHub. Maybe it's only for repo owners? Or maybe it's an option at the moment of merging the pull request? This makes sense to me.

I squashed the commits on my local repo, and git told me to pull first and then push, so I did, and we'll see if that worked.

**Comment by patogit** [comment]
> Also, can you clarify whether this file is generated by script or manually? I think there must be a script that keeps this file up to date when strings are changed or you won't be getting translations for the new strings.

The script to keep the file up to date isn't automated, and it doesn't insert the headers nor the `#, fuzzy` line. To generate the file, I ran the script, and then I added the headers and the `#, fuzzy` line manually.

I looked at the script to see if I could add the headers and fuzzy, but I don't understand the script. As for automating it when commits are made, maybe there's a way to do that with GitHub web hooks? Or maybe there's a different script that meets our needs better?

**Comment by altergui** [comment]
Nice, this squashed commit is just what i had in mind
29ec735c7e
contgratulations!
problem is, you *merged* that commit with the "messy" branch (03f8cc2974), resulting in 2861f04488, which is even messier than the original branch

using git via command line, this is what i would do:
`git checkout -B hotfix/create-en-po-file 29ec735c7e`
`git push -f git://github.com/patogit/lime-packages.git hotfix/create-en-po-file`

that will force a non-fastforward push to your repo, and you'll lose the original 4 commits, but that's exactly what we want :)

**Comment by altergui** [comment]
i don't have enough experience with github to say if it will allow you to do that. in case it doesn't, simply push that nice commit to a new branch and open a new pull request :(

**Comment by patogit** [comment]
> using git via command line, this is what i would do:
`git checkout -B hotfix/create-en-po-file 29ec735c7e`
`git push -f git://github.com/patogit/lime-packages.git hotfix/create-en-po-file`

Done. The only thing I had to change was the protocol of the second command, from `git://` to `https://`.

Does it look the way we want it now?

**Comment by patogit** [comment]
It looks clean here, https://github.com/patogit/lime-packages/commits/hotfix/create-en-po-file , only one commit. I'm not sure what I should have done differently in the first place, but hey, it's done now.

**Comment by altergui** [comment]
i can't find a way in github to see graph the messy branch.
using gitk (`apt install gitk`), it's evident.
but yes, now it's clean :)

**Pull request merged**